### DISCUSSION TITLE
Hi! Just want to check if the Windows install from the website is working properly

### DISCUSSION THREAD

**Discussion by alexpinel** [alexpinel/Dot#2] [Polls]
Some people have reported issues witht the Windows install so I wanted to check if it is a problem with the app itself.

**Comment by jovihava** [comment]
Just installed, the app started fairly quickly, now waiting for the LLM to load. Hoping to put my GPU to work soon.

**Reply by jovihava** [reply]
Aand now it is stuck in 69%, four hours after strating the installation 
![image](https://github.com/user-attachments/assets/eb088f02-23f1-4e2d-8f63-5cf8ba4910a0)
